src/gtest/data/tx-orchard-duplicate-nullifiers.h

```diff
@@ -0,0 +1,4 @@
84fe44418e8de8d8fb5dcfe6323d26d16a88b3a35ea13c690f851502de4388869058f204b7251ced76a4c6137f04261ee71f688fde348fce40b935af1a37e12acfbf4b3f188527a67e35f88f7899d98065ec02a658eacb13896f3567c60e3339d856752d1c81f1a90e1268f3f38fe7af8ebbaa4ec5d5718ddd8328a0c9678e35af6f2abbb8d1d1265b9afbcdae34444f6718c20bd25494f5a7ea05879c6686c40c97859c835f9f3544cb7190ae1bad32be0db4a815286805c0a33756dcaa2c10b93a9676cde7e300a382abfe6aaac42d3f3411eba022454e02841c74aca20bc1ac72482902f1db40df3b732528a70b50d88a019701252cd69e1f209e896c50316a9ef2fae382359212130e53e2f7cf522ac8a6d63cc25ed4af7c016706add77cde98f1b1fca6966b2a23adfa0593bb68bc32defec4b8b38c2941275594ebd0728f7b00e1cf2f7d86153277627b5925f002deac0bb05357c30f9c29cb3563f3e53bcdbf2d6815507067b64f50e765c96a259e9acb17a712506f561ede6930b11ec2e3ae149230a7cb1fdb1a14a07375b4b460eac3fd2c81c39e6035025aeb984080f2c76dadc01039843f0f8b883f36f1acfb3cd6b21e587492b5280cc2f92dd3f7588dccddc2782f06be4086bef6369c631d24975f5a3ede041d0d1ba03d1d2b5cae1e2c8c6b740f79b9219a7a9c090a113ec63f9422cab034ba095cbb66d070c4d39c02b099aa254882717e2a8e55358b1fbb9c254b7639602c16b54e174e67d71b770a9772a5aacbd8cc5bdbe0d4620633fdfe8050dcbcaa798ea38181e0aaaff29ab582c921c20d76f08ddb5b2fd22b8ae776ed5a41ac8603a1e4672bc26f11fab113b33cc0816e03590ccda1545fe17c2e6a7aa9c03cdd8a8e01e12814d5698a75b15fab131e320ba7f8236a5deec63311a4bf16e07e926b1de9158bffb22644da4c31ea9acc6eda5fadd1bd61a65991345fe17ab4e018aebe1791b36f103d0616d0a439942b3fd098c241cb2b882a3d15043d3466129ab38b3ed320fe6f79d627833374d736a2ce0cee0107123f948b07c2d52104306e6910c74ef763777bdd2ede04c7a9420cfe7fca3db7d74bc4e43e319f43afffe39d146f3314af01b23f26b41a37b92dae82fbacf3f9e360ffc1f1c5466e297abb68bd018d83f410a26ffef09ca5d48aad211006010ac66e2961a3d7920f9185b30a93664e19396d7df7b8dc5e0463f0fc3089ab5c2c4a471682642034f9bb1bfccf046449d04a70210bb07c55ef26839dea0fd409e9b92602b1f11a747a5f7d6b88089c6a58c1b0ec298554219084475f05145aedf0284c4b97997911660ee8dd70b1c7d3e8bd15b9022ef54066c9c31362338d1a92092fe00a80f09348cb488416305997403cb4df14a7f9ae156f96717e6b661be83c7df33f14a1f2ec41b8c774a0b0cdaa222efcfaf838f7ba0d7db202a8fc227e869f631fd04afb1ec06aa5511b117af5678edb698d0e4eb7c7f0cc3fe52644a6f8d58f720c0621627fce7099af0789d50cf8e7aea67cd395c17c234ce5a62133525ba528f07f1bed690fdf53af3f806140255f4dd9a5190b469809e67e762d227587f48f29e6d0915d2f5852b0d56001abd5058bd84f4ca989474820a46a515204b81b6fabc39d3f0c3ca196892e1cae1d068f0555f301d84af9c43e9545316d951b132e931bf561ea12f01a20e7b855a385fb6f8bd5e8640702c261c17707864f211ff81ca3e1f3782b04d69e33ee93ec286a1873b82ad20a43e1d79d8fd3fa9820383ba2bf45133d5a621e3145c3c70c0f5e9faddfe65d109735cf60096b868a2ae86feb61a1d2f92a190108b7d31ded1cd3ff4d726c6f3dd68a3fbe50b720273efba99d85b987a2cf1567957f373cd4251f98b632643481c7d089fb3d150f0505355d6412e4d2ddd1ab611da3b03819c524ba043f1b155ef36033497583b4804cd18ce0ddb855ce027e5fa1e3d7e5bdd2793bc5b20cf417887e40761c0e9f46a305fd5098bf5f5d8fcb303ebc52c091c9201f945d2627f1184c3aae5e8420f874b52844d35e6392a767e19f2996f18050bcea6fd413adf42281bad5229f0544bb3bb339e3e894f8f21ee22f";
```

Comment:
I verified that this does indeed contain two Orchard actions that use the same nullifier (and a valid proof, indicating the actions double-spend the same note).